application/controllers/RegisterController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+// use Google\Service\CloudSearch\Resource\Query;
+require_once __DIR__ . '/../models/QueryModel.php';
```

Comment:
We have invoked session start multiple times in this file. Please make sure they are handled properly